Formula/no-more-secrets.rb

```diff
@@ -0,0 +1,16 @@
+class NoMoreSecrets < Formula
+  desc 'Recreates the SETEC ASTRONOMY effect from "Sneakers"'
```

Comment:
I switched it to single-quotes-in-double-quotes in 60eb33f7456dd7c605a8ca56929c98ddd05d8545, since that seems to be preferred and it still reads fine as both code and `brew info no-more-secrets` output.